rpc/transport/transport.go

```diff
@@ -142,7 +165,10 @@ func (s *transport) NewMessage(ctx context.Context) (_ rpccp.Message, send func(
 		return err
 	}
 
-	return rmsg, send, func() { msg.Reset(nil) }, nil
+	return rmsg, send, func() {
+		msg.Reset(nil)
```

Comment:
We're still allocating a new `Message` on each call to `NewMessage`, which accounts for ~5% of allocation space in the post-test.

Can we reuse these as well?  (Separate PR is fine.)

Reply:
We could do that. My plan was basically to do things in the order the profiler tells me until we're satisfied.

Reply:
Sounds good.  Resolving.